c-api/tests/c_consumer.c

```
@@ -83,6 +83,86 @@ int main(void) {
+        unsigned char *scomp = (unsigned char *)malloc(bound + 64);
+        if (!sbuf || !scomp) return 2;
+        size_t scomp_len = 0;
+        const size_t chunk = 64 * 1024;
+        for (size_t off = 0; off < n; off += chunk) {
+            size_t len = n - off < chunk ? n - off : chunk;
+            ZSTD_inBuffer inb = {input + off, len, 0};
+            while (inb.pos < inb.size) {
+                ZSTD_outBuffer outb = {sbuf, out_cap, 0};
+                size_t rc = ZSTD_compressStream2(zcs, &outb, &inb, ZSTD_e_continue);
+                if (ZSTD_isError(rc)) return 26;
+                memcpy(scomp + scomp_len, sbuf, outb.pos);
```

Cppcheck (2.20.0)

[error] Memory is allocated but not initialized

(uninitdata)